src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustDataDto.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dto.trust;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class TrustDataDto {
+
+    @JsonProperty("trust_id")
+    private String trustId;
+
+    @JsonProperty("trust_name")
+    private String trustName;
+
+    @JsonProperty("creation_date")
+    private LocalDate creationDate;
+
+    @JsonProperty("unable_to_obtain_all_trust_info")
+    private Boolean unableToObtainAllTrustInfo;
+
+    @JsonProperty("INDIVIDUAL")
```

Comment:
See previous comment regarding the capitalisation of these names (and I also think they should be plurals).